core/src/main/java/org/elasticsearch/transport/netty/NettyTransportChannel.java

```diff
@@ -134,6 +141,14 @@ public void sendResponse(Throwable error) throws IOException {
         transportServiceAdapter.onResponseSent(requestId, action, error);
     }
 
+    private void close() {
```

Comment:
> That's a pure implementation detail.

That's exactly what we are discussing here though? The semantics of *this* implementation.

> From the perspective of the channel it is still "closed", but if we really have to do this, then I'd more lean towards release.

This was never an issue though? It was added as part of this PR to release the tracked bytes (also added in this PR).